pkg/processor/build/builder.go

```diff
@@ -1363,16 +1363,52 @@ func (b *Builder) commandsToDirectives(commands []string) (map[string][]function
 	currentDirective := "preCopy"
 
 	// iterate over commands
-	for _, command := range commands {
+	for i := 0; i < len(commands); i++ {
```

Comment:
why not using `for i, command := range commands {` ?

Reply:
because we append stuff we may skip commands if there's a multiline